packages/doura/src/core/modelManager.ts

```diff
@@ -208,6 +210,14 @@ class ModelManagerInternal implements ModelManager {
     modelInstance.subscribe(this._onModelChange)
 
     this._models.set(name, modelInstance)
+    if (module.hot) {
+      if (!isArray((model as HmrModel)[CLEAR_CACHE])) {
+        ;(model as HmrModel)[CLEAR_CACHE] = []
```

Comment:
this won't work when there are multiple model managers

Reply:
diff model managers has diff `this._models`, this._models is base on it's own model managers